### ISSUE TITLE
Support for HL-DT-ST BD-RE BU40N

### ISSUE THREAD

**Issue by headhunter45** [saramibreak/DiscImageCreator#181]
**Is your feature request related to a problem? Please describe.**
I'm trying to read a playstation cd and I'm getting a bunch of C2 errors and then a probably more serious "logical block address out of range" error that ends processing. I know this is an unsupported drive, but it's listed as "maybe works" on the redump wiki. http://wiki.redump.org/index.php?title=List_of_Unsupported_Drives . I'm trying to find out if this is an error with my disk/firmware or just that it's not going to work because it's not supported. Also if it's not supported what can I do to get it supported.

I'm using the 1.03 MK firmware from makemkv.com to backup 4k blu-rays. I've got 3 of these drives and I'm willing to try flashing other firmware to one of them.

**Describe the solution you'd like**
I'd like someone to tell me if I should give up or if there is something I can do to help. If there's some obvious path like testing the drive and writing some code to add support I'm willing to try.

**Describe alternatives you've considered**
* Looked for some of the supported plextor drives on online shopping sites.
* I haven't tried backing up any other media yet.

**Additional context**
The command I'm running is `DiscImageCreator.exe cd e FinalFantasyTactics_psx.bin 40 /q /c2 /nl`. I'm entering the +6 offset from http://www.accuraterip.com/driveoffsets.htm and the list of unsupported drives page on redump.

These are the errors/warnings at the end of the run.
```
Creating .scm (LBA) 175179/230150 LBA[175180], LBA translation to SCM address in hex[0x188efa40], LBA in C2 file[0x311df48] Detected C2 error 27 bit
Creating .scm (LBA) 179643/230150 LBA[179644], LBA translation to SCM address in hex[0x192f2f40], LBA in C2 file[0x325e5e8] Detected C2 error 70 bit
Creating .scm (LBA) 179644/230150 LBA[179645], LBA translation to SCM address in hex[0x192f3870], LBA in C2 file[0x325e70e] Detected C2 error 16 bit
Creating .scm (LBA) 181218/230150 LBA[181219], LBA translation to SCM address in hex[0x1967b590], LBA in C2 file[0x32cf6b2] Detected C2 error 5 bit
LBA[230151, 0x38307]: [F:ProcessReadCD][L:336]
        Opcode: 0xbe
        ScsiStatus: 0x02 = CHECK_CONDITION
        SenseData Key-Asc-Ascq: 05-21-00 = ILLEGAL_REQUEST - LOGICAL BLOCK ADDRESS OUT OF RANGE
Creating .scm (LBA) 230150/230150
Need to reread sector: 112227 rereading times: 4000/4000
bad all. need to reread more
EndTime: 2023-02-24T00:18:01-0800
```

**Comment by saramibreak** [comment]
Generally, C2 error is the disc problem. Please get another one.

**Comment by saramibreak** [comment]
I fixed/changed the c2 error rereading in #196.
If you are interested in it, try it please.
